draft-ietf-tls-tls13.md

```diff
@@ -2189,11 +2214,18 @@ be taken into account when designing new extensions:
 %%% Version Extension
 
        struct {
-           ProtocolVersion versions<2..254>;
+           select (Handshake.msg_type) {
+               case client_hello:
+                    ProtocolVersion versions<2..254>;
+                    
+               case server_hello:
+                    ProtocolVersion selected_version;
```

Comment:
There is probably some editorial (?) work to be done where we still talk about HRR all over the place (in the extensions table), etc., but there is no longer a hello_retry_request message type for the actual protocol elements.